src/docs/guides/typed-components.md

```diff
@@ -30,36 +30,5 @@ button.fill = 'outline';
 
 **IMPORTANT**: always use the `HTML{}Element` interfaces in order to hold references to components.
 
+TODO: Put a redirect/link here for the links I just deleted
```

Comment:
I need to get these redirects set up still

Reply:
I went for a 'cheap' way to do this in https://github.com/ionic-team/stencil-site/pull/832/commits/9638d586e3da7209de33d8fecd983439aced85fc - I don't think Vercel supports redirects that use document fragments. So while I could get redirects up to the 'properties' page, it's not a great experience trying to find the section from this page that maps to the properties page.